lib/exports/colors-new/__snapshots__/colors.less.test.ts.snap

```diff
@@ -0,0 +1,752 @@
+    --blue-new-300: hsl(210, 78%, 76%);
+    --blue-new-400: hsl(210, 70%, 48%);
+    --blue-new-500: hsl(210, 75%, 36%);
+    --blue-new-600: hsl(210, 80%, 23%);
+    --green-new-100: hsl(148, 35%, 95%);
+    --green-new-200: hsl(148, 35%, 88%);
+    --green-new-300: hsl(148, 35%, 69%);
+    --green-new-400: hsl(148, 70%, 31%);
+    --green-new-500: hsl(148, 75%, 22%);
+    --green-new-600: hsl(148, 75%, 15%);
+    --red-new-100: hsl(0, 78%, 96%);
+    --red-new-200: hsl(0, 70%, 93%);
+    --red-new-300: hsl(0, 65%, 76%);
+    --red-new-400: hsl(0, 60%, 49%);
+    --red-new-500: hsl(0, 65%, 37%);
+    --red-new-600: hsl(0, 65%, 24%);
+    --yellow-new-100: hsl(43, 80%, 96%);
+    --yellow-new-200: hsl(43, 80%, 88%);
+    --yellow-new-300: hsl(43, 85%, 72%);
+    --yellow-new-400: hsl(43, 85%, 50%);
+    --yellow-new-500: hsl(43, 85%, 33%);
+    --yellow-new-600: hsl(43, 84%, 18%);
+    --gold-new-100: hsl(46, 100%, 91%);
+    --gold-new-200: hsl(46, 100%, 74%);
+    --gold-new-300: hsl(45, 100%, 42%);
+    --gold-new-400: hsl(46, 92%, 26%);
+    --silver-new-100: hsl(0, 0%, 95%);
+    --silver-new-200: hsl(0, 0%, 84%);
+    --silver-new-300: hsl(210, 5%, 68%);
+    --silver-new-400: hsl(210, 2%, 40%);
+    --bronze-new-100: hsl(28, 40%, 92%);
+    --bronze-new-200: hsl(30, 47%, 83%);
+    --bronze-new-300: hsl(28, 43%, 65%);
+    --bronze-new-400: hsl(28, 43%, 39%);
+    --bc-new-lightest: var(--black-new-100);
+    --bc-new-lighter: var(--black-new-150);
+    --bc-new-light: var(--black-new-200);
+    --bc-new-medium: var(--black-new-225);
+    --bc-new-dark: var(--black-new-250);
+    --bc-new-darker: var(--black-new-300);
+    --bs-new-sm: 0 1px 2px hsla(0, 0%, 0%, 0.05), 0 1px 4px hsla(0, 0%, 0%, 0.05), 0 2px 8px hsla(0, 0%, 0%, 0.05);
+    --bs-new-md: 0 1px 3px hsla(0, 0%, 0%, 0.06), 0 2px 6px hsla(0, 0%, 0%, 0.06), 0 3px 8px hsla(0, 0%, 0%, 0.09);
+    --bs-new-lg: 0 1px 4px hsla(0, 0%, 0%, 0.09), 0 3px 8px hsla(0, 0%, 0%, 0.09), 0 4px 13px hsla(0, 0%, 0%, 0.13);
+    --bs-new-xl: 0 10px 24px hsla(0, 0%, 0%, 0.05), 0 20px 48px hsla(0, 0%, 0%, 0.05), 0 1px 4px hsla(0, 0%, 0%, 0.1);
+    --focus-new: hsla(206, 100%, 40%, 0.9);
+    --focus-new-success: hsla(140, 40%, 75%, 0.4);
+    --focus-new-warning: hsla(47, 79%, 58%, 0.4);
+    --focus-new-error: hsla(358, 62%, 47%, 0.15);
+    --focus-new-muted: hsla(210, 8%, 15%, 0.1);
+    --highlight-new-addition: var(--green-700);
+    --highlight-new-attribute: hsl(206, 98.5%, 29%);
+    --highlight-new-bg: hsl(0, 0%, 96.5%);
+    --highlight-new-color: var(--black-750);
+    --highlight-new-comment: hsl(210, 8%, 43.5%);
+    --highlight-new-deletion: var(--red-600);
+    --highlight-new-keyword: hsl(206, 98.5%, 29%);
+    --highlight-new-literal: hsl(27, 99%, 36%);
+    --highlight-new-namespace: hsl(27, 99%, 36%);
+    --highlight-new-punctuation: var(--black-600);
+    --highlight-new-symbol: hsl(306, 43%, 35%);
+    --highlight-new-variable: hsl(80, 80.5%, 26.5%);
```

Comment:
> **Note**
> `--highlight-*` variables referencing legacy colors need to be remapped.
> This should be done in a follow-up PR.

Reply:
I wrote a comment about this in my review. I guess this should be done in a follow up PR because we need design to confirm the mapping `old` -> `new` color?